crates/common/fork_choice/src/store.rs

```diff
@@ -134,6 +134,56 @@ impl Store {
         Ok(false)
     }
 
+    /// Retrieve a filtered block tree from ``store``, only returning branches
+    /// whose leaf state's justified/finalized info agrees with that in ``store``.
+    pub fn get_filtered_block_tree(&self) -> anyhow::Result<HashMap<B256, BeaconBlock>> {
+        let base = self.db.justified_checkpoint_provider().get()?.root;
+
+        let mut blocks: HashMap<B256, BeaconBlock> = HashMap::default();
+
+        self.filter_block_tree(base, &mut blocks)?;
+
+        Ok(blocks)
+    }
```

Comment:
```suggestion
    pub fn get_filtered_block_tree(&self) -> anyhow::Result<HashMap<B256, BeaconBlock>> {
        let base = self.db.justified_checkpoint_provider().get()?.root;
        let mut blocks = HashMap::default();
        self.filter_block_tree(base, &mut blocks)?;
        Ok(blocks)
    }
```

we are fine without the type decoration and the code is simple enough where spaces aren't needed